src/script/components/TitleBar/TitleBar.tsx

```diff
@@ -24,7 +24,7 @@ import {amplify} from 'amplify';
 import cx from 'classnames';
 import {container} from 'tsyringe';
 
-import {IconButton, IconButtonVariant, useMatchMedia} from '@wireapp/react-ui-kit';
+import {IconButton, IconButtonVariant, QUERY, useMatchMedia} from '@wireapp/react-ui-kit';
```

Comment:
I personally dont love the name QUERY, since my developer brain connects database requests or api requests with that term. 

I would love descriptive names, in this case, MEDIA_QUERY (since we will probably also work with container queries in the future).

Reply:
https://github.com/wireapp/wire-web-packages/blob/main/packages/react-ui-kit/src/mediaQueries.ts
I agree with you but thats how it is setup in `react-ui-kit`. Let's discuss it with the team and find the better option.